sonaranalyzer-dotnet/its/ReviewDiffs/ReviewDiffs.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <LangVersion>8.0</LangVersion>
     <SonarQubeExclude>true</SonarQubeExclude>
+    <ProjectGuid>{7e3adcd6-01d7-44e2-b12f-c485c35c8a9c}</ProjectGuid>
```

Comment:
`sonnarscanner` is complaining about missing project guids when `dotnet build` is executed.